### PR TITLE
[zh] sync auth docs

### PR DESCRIPTION
**Pull request by chenxuc**
related: #29327

Sync Chinese for following:
```
1	1	content/zh/docs/reference/access-authn-authz/abac.md
11	2	content/zh/docs/reference/access-authn-authz/authorization.md
4	9	content/zh/docs/reference/access-authn-authz/service-accounts-admin.md (already synced. No change in this PR)
14	0	content/zh/docs/reference/access-authn-authz/admission-controllers.md
26	3	content/zh/docs/reference/access-authn-authz/rbac.md
```

### CODE REVIEW

File: content/zh/docs/reference/access-authn-authz/abac.md [hunk 1]
```diff
@@ -73,7 +73,7 @@ properties:
 
 基于 `ABAC` 模式，可以这样指定策略文件 `--authorization-policy-file=SOME_FILENAME`。
 
-此文件格式是 [JSON Lines](https://jsonlines.org/)，不应存在封闭的列表或映射，每行一个映射。
+此文件格式是 [JSON Lines](https://jsonlines.org/)，不应存在封闭的列表或映射，每行只有一个映射。
```
Comment (tengqm) [inline content/zh/docs/reference/access-authn-authz/abac.md:76]: 这里的 enclosing 可以理解为“最外层的”

File: content/zh/docs/reference/access-authn-authz/admission-controllers.md [hunk 2]
```diff
@@ -1171,8 +1171,28 @@ PodNodeSelector 允许 Pod 强制在特定标签的节点上运行。
...
+{{< feature-state for_k8s_version="v1.22" state="alpha" >}}
+
+<!--
+This is the replacement for the deprecated [PodSecurityPolicy](#podsecuritypolicy) admission controller
+defined in the next section. This admission controller acts on creation and modification of the pod and
+determines if it should be admitted based on the requested security context and the 
+[Pod Security Standards](/docs/concepts/security/pod-security-standards/).
+
+See the [Pod Security Admission documentation](/docs/concepts/security/pod-security-admission/)
+for more information.
```
Comment (tengqm) [inline content/zh/docs/reference/access-authn-authz/admission-controllers.md:1185]: 未译

File: content/zh/docs/reference/access-authn-authz/authorization.md [hunk 3]
```diff
@@ -352,36 +383,64 @@ so an earlier module has higher priority to allow or deny a request.
...
+Details of how these can be misused are documented in [escalation paths](/docs/reference/access-authn-authz/authorization/#escalation-paths)
 -->
-系统管理员在授予对 Pod 创建的访问权限时要小心。
-被授予在名字空间中创建 Pod（或创建 Pod 的控制器）的权限的用户可以：
-读取名字空间中的所有 Secret；读取名字空间中的所有 ConfigMap；
-并模拟名字空间中的任意服务账号并执行账号可以执行的任何操作。
-无论采用何种鉴权方式，这都适用。
+系统管理员在授予对工作负载的创建或编辑的权限时要小心。
+关于这些权限如何被误用的详细信息请参阅
+[提升途径](/zh/docs/reference/access-authn-authz/authorization/#escalation-paths)
```
Comment (tengqm) [inline content/zh/docs/reference/access-authn-authz/authorization.md:404]: 这个目标锚点就在本文件中，所以前面的路径看起来不必要

File: content/zh/docs/reference/access-authn-authz/rbac.md [hunk 4]
```diff
@@ -1068,15 +1082,21 @@ It does not allow write access to resource quota or to the namespace itself.
...
+the namespace. This role also does not allow write access to Endpoints in
+clusters created using Kubernetes v1.22+. More information is available in the
+["Write Access for Endpoints" section](#write-access-for-endpoints).
 -->
 允许对名字空间的大多数对象进行读/写操作。
+
 它不允许查看或者修改角色或者角色绑定。
 不过，此角色可以访问 Secret，以名字空间中任何 ServiceAccount 的身份运行 Pods，
 所以可以用来了解名字空间内所有服务账户的 API 访问级别。
+此角色也不允许对 Kubernetes v1.22+ 创建的 Endpoint 进行写操作。
```
Comment (tengqm) [inline content/zh/docs/reference/access-authn-authz/rbac.md:1098]: Endpoints 是复数，不存在 Endpoint 资源

File: content/zh/docs/reference/access-authn-authz/rbac.md [hunk 5]
```diff
@@ -1068,15 +1082,21 @@ It does not allow write access to resource quota or to the namespace itself.
...
+更多信息参阅[“Endpoint 写操作”小节](#write-access-for-endpoints)。
```
Comment (tengqm) [inline content/zh/docs/reference/access-authn-authz/rbac.md:1099]: 同上

File: content/zh/docs/reference/access-authn-authz/rbac.md [hunk 6]
```diff
@@ -1957,6 +1977,40 @@ In order from most secure to least secure, the approaches are:
...
+subject to this change. The [CVE
+announcement](https://github.com/kubernetes/kubernetes/issues/103675) includes
+guidance for restricting this access in existing clusters.
+
+If you want new clusters to retain this level of access in the aggregated roles,
+you can create the following ClusterRole:
+
+{{< codenew file="access/endpoints-aggregated.yaml" >}}
+-->
+## Endpoint 写权限 {#write-access-for-endpoints}
```
Comment (tengqm) [inline content/zh/docs/reference/access-authn-authz/rbac.md:1999]: Endpoints，下同

File: content/zh/docs/reference/access-authn-authz/rbac.md [hunk 7]
```diff
@@ -1957,6 +1977,40 @@ In order from most secure to least secure, the approaches are:
...
+作为 [CVE-2021-25740](https://github.com/kubernetes/kubernetes/issues/103675) 的缓解措施，
+此访问权限不包含在 Kubernetes 1.22 以及更高版本集群的聚合角色里。
+
+升级到 Kubernetes v1.22 版本的现有集群不会包括此变化。
+[CVE 公告](https://github.com/kubernetes/kubernetes/issues/103675)
+包含了在现有集群里限制此访问权限的指引。
+
+如果你希望在新集群的聚合角色里保留此访问权限，你可以创建下面的 ClusterRole：
+
+{{< codenew file="access/endpoints-aggregated.yaml" >}}
```
Comment (tengqm) [inline content/zh/docs/reference/access-authn-authz/rbac.md:2012]: 这个文件是不是 zh 下面还没有？

File: content/zh/docs/reference/access-authn-authz/rbac.md (hunk 4 above)
Comment (howieyuen) [reply content/zh/docs/reference/access-authn-authz/rbac.md:1098]: > 

我检查了下k8s所有的内置API，只有endpoints的kind是复数，其他都是单数，:curious

**Comment by jihoon-seo** [comment]
[Netlify build fails](https://app.netlify.com/sites/kubernetes-io-main-staging/deploys/61a37e787bd2770008c12185#L77).
> `content/zh/examples/access/endpoints-aggregated.yaml`: no such file or directory

My suggestion:
Copy `content/en/examples/access/endpoints-aggregated.yaml` to `content/zh/examples/access/endpoints-aggregated.yaml`,
and optionally translate it if needed.

@chenxuc PTAL!

**Review by tanjunchen — COMMENTED** [review]
/LGTM
/retest

**Comment by tengqm** [comment]
Looks like we may need to resubmit this?

**Comment by chenxuc** [comment]
I resubmit the changes in a new PR #30656

**Pull request merged**